### PR TITLE
upgrade 3Box

### PR DESCRIPTION
**Pull request by dkent600**
Resolves: https://github.com/daostack/alchemy/issues/1668

Messing around with this, the basic 3box functionality works, but it is very fragile.  I'd say just a little more fragile than on Alchemy 1.0, which also is also fragile.  By "fragile" I mean is is easy to find bugs, particularly when switching Metamask accounts in mid-stream.  I think we could take this PR just to solve the slug problem, without much cost, but ought to give the 3Box functionality some good solid testing and fix the bugs.  Having seen the code, I suspect some of it could use some refactoring (such as I have done in the 3Box comments PR).

### CODE REVIEW

**Comment by dkent600** [comment]
Somehow I haven't yet figured out, the inclusion of the 3box package appears to be causing the react-sticknode component to misbehave, causing the layout problem below and causing two unit tests to fail:

image

**Comment by dkent600** [comment]
To deal with [this problem](https://github.com/daostack/alchemy/pull/1947#issuecomment-657699062) I had to remove the `sticky` component in DaoPluginsPage, which is a desired refactor in any case, and rewrite the sticky behavior using css from scratch.

File: src/components/Dao/DaoPluginsPage.scss [hunk 1]
```diff
@@ -1,13 +1,64 @@
...
-    width: 100%;
-    padding-top: 30px;
-    background-color: rgba(229, 235, 242, 1);
-    margin: 0;
+  .paddingTop {
+    position: sticky;
+    top: 50px;
+    height: 62px;
+    z-index: 2;
+    background-color: #e5ebf2;
```
Comment (roienatan) [inline src/components/Dao/DaoPluginsPage.scss:12]: Please use global variables for colors from `global-variables.scss`.
I know that in many places it's hard-coded but it's a pattern we should eliminate.

Comment (roienatan) [reply src/components/Dao/DaoPluginsPage.scss:12]: Also in other places.

Comment (dkent600) [reply src/components/Dao/DaoPluginsPage.scss:12]: I totally agree

Comment (dkent600) [reply src/components/Dao/DaoPluginsPage.scss:12]: I converted to color and border variables for everything in this file, plus all other files that use the same values.  I didn't address the naming convention of these variables, which can be addressed at a future time.

Comment (roienatan) [reply src/components/Dao/DaoPluginsPage.scss]: Looks good

File: src/arc.ts [hunk 2]
```diff
@@ -154,6 +154,7 @@ export async function initializeArc(provider?: Web3Provider): Promise<boolean> {
     } else {
       arc = new Arc(arcSettings);
     }
+    arc.web3.pollingInterval = 60000;
```
Comment (orenyodfat) [inline src/arc.ts:157]: this already been set https://github.com/daostack/alchemy/blob/dev-2/src/arc.ts#L157

File: src/components/Account/Account.scss [hunk 3]
```diff
@@ -331,8 +331,8 @@ input {
 }
 
 .submitButton:disabled {
-  background-color: rgba(200, 200, 200, 0.5);
-  border: 1px solid rgba(200, 200, 200, 1);
+  background-color: $disabled-button-color;
```
Comment (orenyodfat) [inline src/components/Account/Account.scss:334]: how this related to 3box ?

File: src/components/Dao/DaoLandingPage.scss [hunk 4]
```diff
@@ -15,7 +15,7 @@
   .headerText {
     font-size: 28px;
     font-weight: bold;
-    color: rgba(104, 155, 214, 1);
+    color: $title-color;
```
Comment (orenyodfat) [inline src/components/Dao/DaoLandingPage.scss:18]: how this related to 3box ?

File: src/components/Dao/DaoPluginsPage.tsx [hunk 5]
```diff
@@ -108,24 +106,26 @@ class DaoPluginsPage extends React.Component<IProps, null> {
...
-              data-test-id="createProposal"
-              href="#!"
-              onClick={pluginManagerActive ? this.handleNewProposal(pluginManager.id) : null}
-              >
-                Add a Plugin
-              </a>
-            </TrainingTooltip>
-            : ""}
-        </Sticky>
+        <div className={css.paddingTop}>&nbsp;</div>
```
Comment (orenyodfat) [inline src/components/Dao/DaoPluginsPage.tsx:109]: does this change related to 3box ?

File: src/components/Account/Account.scss (hunk 3 above)
Comment (dkent600) [reply src/components/Account/Account.scss:334]: https://github.com/daostack/alchemy/pull/1947#issuecomment-658451101

File: src/components/Dao/DaoLandingPage.scss (hunk 4 above)
Comment (dkent600) [reply src/components/Dao/DaoLandingPage.scss:18]: https://github.com/daostack/alchemy/pull/1947#issuecomment-658451101

File: src/components/Dao/DaoPluginsPage.tsx (hunk 5 above)
Comment (dkent600) [reply src/components/Dao/DaoPluginsPage.tsx:109]: https://github.com/daostack/alchemy/pull/1947#issuecomment-658451101

File: src/components/Dao/DaoLandingPage.scss (hunk 4 above)
Comment (dkent600) [reply src/components/Dao/DaoLandingPage.scss:18]: @roienatan asked to use color variables, a good request.

**Pull request merged**